src/components/input/input.spec.js

```diff
@@ -4,9 +4,9 @@ describe('md-input-container directive', function() {
   function setup(attrs, isForm) {
     var container;
     inject(function($rootScope, $compile) {
-      container = $compile((isForm ? '<form>' : '') + 
-          '<md-input-container><input ' +(attrs||'')+ '><label></label></md-input-container>' +
-          (isForm ? '<form>' : ''))($rootScope);
+      container = $compile((isForm ? '<form>' : '') +
```

Comment:
Just a heads up to watch for whitespace changes in commits...we try to avoid them.